store/src/java/com/zimbra/cs/mailbox/calendar/ZAttendee.java

```diff
@@ -142,10 +142,10 @@ public ZAttendee(ZProperty prop) {
         setRole(prop.paramVal(ICalTok.ROLE, null));
         setPartStat(prop.paramVal(ICalTok.PARTSTAT, null));
 
-        String rsvpStr = prop.paramVal(ICalTok.RSVP, "FALSE");
-        boolean rsvp = false;
-        if (rsvpStr.equalsIgnoreCase("TRUE"))
-            rsvp = true;
+        String rsvpStr = prop.paramVal(ICalTok.RSVP, "TRUE");
+        boolean rsvp = true;
```

Comment:
this is setting rsvp to true by default. did you test by sending appointment with rsvp=false?